### DISCUSSION TITLE
Question: How are you using Flipt?

### DISCUSSION THREAD

**Discussion by markphelps** [flipt-io/flipt#198] [General]
👋 Hi, I'm Mark, creator of Flipt.

I was wondering if anyone was willing to share how they or their company was making use of Flipt (in production or otherwise)?

Since Flipt doesn't collect any telemetry data (on principle), it's near impossible for me to find out how it's actually being used. 

Still, I'd like to know if people are finding this project useful and if there is anything I can do to help you be successful with whatever you are using Flipt for.

If you're willing to share, please comment below.

**A couple questions I had in mind were:**

1. How do you or your company use Flipt? What are your usecases? Feel free to provide links to your site or product
1. If you aren't using Flipt in production, what is stopping you?
1. How did you find out about Flipt?
1. How has your experience been so far with Flipt? Is there anything that would help improve your experience?
1. Have you used any other feature flag software before using Flipt? If so, what? Why did you choose Flipt over others?

Thank you in advance for any feedback you provide!

Cheers 🍻 
Mark

**Comment by wsargent** [comment]
I'm not using flipt (yet) but I am trying to get more familiar with feature flag systems.  As a developer, all I really care about is making it reproducible as "not a service" so running in a docker container is ideal, and the user feature of flipt looks interesting because I should be able to run https://tersesystems.com/blog/2019/07/22/targeted-diagnostic-logging-in-production/ against it.

**Comment by markphelps** [comment]
@wsargent this was a great read thank you! Would love to know more if you end up using Flipt for this!

**Comment by twgraham** [comment]
More than happy to share 👍 

**How do you or your company use Flipt? What are your usecases? Feel free to provide links to your site or product.**

We're using it over at [InfoPoint](https://infopoint.com.au). I was part way through designing an in house feature management service when I found out about flipt. It looked well designed, with clear intentions. We use it for managing various API endpoint access, and enabling/disabling/changing various parts of our UI. This is usually based around subscription levels, but could also be due to custom requirements of customers, or canary testing new features.

**How did you find out about Flipt?**

I'm pretty sure it was HackerNews... though it could've been a follow up google for inspiration.

**How has your experience been so far with Flipt? Is there anything that would help improve your experience**

Great, though I haven't had to manage too much since it was deployed (though that may change). Hard to say for improvements 

**Have you used any other feature flag software before using Flipt? If so, what? Why did you choose Flipt over others**

No not really - just never had to manage that part of a system.

---

If you have any other questions feel free to message me. ✉️
